src/test/java/org/folio/des/scheduling/quartz/EdifactExportJobSchedulerTest.java

```diff
@@ -160,6 +162,43 @@ void testJobRemovedForRemovedConfiguration() throws SchedulerException {
       () -> assertEquals(0, scheduler.getJobKeys(GroupMatcher.anyJobGroup()).size()));
   }
 
+  @Test
+  void testDeleteJobGroup() throws SchedulerException {
+    ExportConfig config = getExportConfig();
+    edifactExportJobScheduler.scheduleExportJob(config);
+
+    scheduledJobsRemover.deleteJob(TENANT);
+
+    var jobKeys = scheduler.getJobKeys(GroupMatcher.anyJobGroup());
+    assertEquals(0, jobKeys.size());
+  }
+
+  @Test
+  void testNotDeleteIfJobDontExists() throws SchedulerException {
+    ExportConfig config = getExportConfig();
+    edifactExportJobScheduler.scheduleExportJob(config);
+
+    scheduledJobsRemover.deleteJob("");
+
+    var jobKeys = scheduler.getJobKeys(GroupMatcher.anyJobGroup());
+    assertEquals(1, jobKeys.size());
+  }
+
+  @Test
+  void testNotDeleteOtherJobGroup() throws SchedulerException {
+    ExportConfig config = getExportConfig();
+    ExportConfig otherConfig = getExportConfig();
+    otherConfig.setTenant("tenant-other");
```

Comment:
could you please a test(or update this) to make sure that if there're several jobs for tenant, all of them are deleted?

Reply:
Test is improved